src/utils/plot.py

```diff
@@ -0,0 +1,40 @@
+import seaborn as sns
+import matplotlib as plt
+import numpy as np
+
+def plot_training_curve(path):
+    """ Plots the training curve for a model run, given the csv files
+    containing the train/validation accuracy/loss.
+
+    Args:
+        path: The base path of the csv files produced during training
+    """
+    train_acc = np.loadtxt("./csv/{}_train_acc.csv".format(path))
+    val_acc = np.loadtxt("./csv/{}_val_acc.csv".format(path))
+    train_loss = np.loadtxt("./csv/{}_train_loss.csv".format(path))
+    val_loss = np.loadtxt("./csv/{}_val_loss.csv".format(path))
+    plt.title("Train vs Validation Accuracy")
+    n = len(train_acc) # number of epochs
+    plt.plot(range(1,n+1), train_acc, label="Train")
+    plt.plot(range(1,n+1), val_acc, label="Validation")
+    plt.xlabel("Epoch")
+    plt.ylabel("Accuracy")
+    plt.legend(loc='best')
+    plt.show()
+    plt.title("Train vs Validation Loss")
+    plt.plot(range(1,n+1), train_loss, label="Train")
+    plt.plot(range(1,n+1), val_loss, label="Validation")
+    plt.xlabel("Epoch")
+    plt.ylabel("Loss")
+    plt.legend(loc='best')
+    plt.show()
+
+def plot_confusion_matrix(model_path, classes):
```

Comment:
How does this ./csv path work? Is this specific to your local machine or does it work with colab?

I also think it might be useful to have this function create the confusion matrix as well. We could add as parameters the prediction and label arrays. Maybe that's for the future, but good to keep in mind.